cmd/state-tray/internal/menu/projects.go

```diff
@@ -54,7 +55,14 @@ func (u *LocalProjectsUpdater) Update(projects []*graph.Project) {
 		u.items = append(u.items, &localProjectsMenuItem{mitem, "", "", cb, make(chan struct{})})
 	}
 
-	u.startEventLoops()
+	stateApp, err := installation.NewAppInfo(installation.StateApp)
+	if err != nil {
+		return locale.WrapError(err, "err_state_info")
+	}
```

Comment:
Nit: Feels more like something to set during construction rather than fed into the Update call.